systemtest/src/test/java/io/strimzi/systemtest/rollingupdate/KafkaRollerST.java

```diff
@@ -120,6 +116,7 @@ void testKafkaRollsWhenTopicIsUnderReplicated(ExtensionContext extensionContext)
         List<Event> events = kubeClient(namespaceName).listEventsByResourceUid(uid);
         assertThat(events, hasAllOfReasons(Scheduled, Pulled, Created, Started));
 
+        // TODO scaling down of Kafka Cluster (with 4 replicas which has KafkaTopic with 4 replicas) can be forbidden in a future due to would-be Topic under-replication
```

Comment:
I guess adding some GH issue here to track it down "in the future" would be great